MozillaFoundation/foundation.mozilla.org
# updated hardcoded RSS url to relative url

# Description



Link to sample test page: https://foundation-s-remove-har-5dcefy.herokuapp.com/blog/rss
Related PRs/issues: #11830 , [TP1-94](https://mozilla-hub.atlassian.net/browse/TP1-94)

As mentioned in #11830, this PR updates the `link` and `feed_url` variables to relative URLs instead of hardcoding them.

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-542)

## Steps to test

1. Open an RSS reader of your choice
2. Save the feed coming from https://foundation-s-remove-har-5dcefy.herokuapp.com/blog/rss
3. If the feed is working as expected (showing pages set up by the factory) testing is complete!
